libsolidity/inlineasm/AsmAnalysis.cpp

```diff
@@ -349,6 +335,25 @@ bool AsmAnalyzer::operator()(Block const& _block)
 	return success;
 }
 
+bool AsmAnalyzer::expectExpression(Statement const& _statement)
+{
+	bool success = true;
+	int const initialHeight = m_stackHeight;
+	if (!boost::apply_visitor(*this, _statement))
+		success = false;
+	if (m_stackHeight - initialHeight != 1)
+	{
+		m_errorReporter.typeError(
+			locationOf(_statement),
+			"Expected instruction(s) to deposit one item to the stack but did deposit " +
```

Comment:
Seems like the text is not checked against in the tests, so don't mind if it is `one` or `1`, but would be nice to change `instruction(s)` to `expression`.